airbyte-config/models/src/main/java/io/airbyte/config/AirbyteProtocolConverters.java

```diff
@@ -77,9 +81,31 @@ public static Schema toSchema(AirbyteCatalog catalog) {
           .withName(airbyteStream.getName())
           .withFields(list.stream().map(item -> new Field()
               .withName(item.getKey())
-              .withDataType(DataType.valueOf(item.getValue().get("type").asText().toUpperCase()))
+              .withDataType(singerTypesToDataType(item.getValue().get("type")))
```

Comment:
Why do we have the word singer in a class that is AB specific?

Reply:
see comment. purely a name issue. not actually singer specific. will fix tha name. https://github.com/airbytehq/airbyte/pull/609/files#r507447435